customer-churn/README.md

```diff
@@ -1,19 +1,19 @@
-# Predicting whether the customer will churn or not before they even did it
+# Will they stay or will they go? Building a Customer Loyalty Predictor
 
 [![PyPI - Python Version](https://img.shields.io/pypi/pyversions/zenml)](https://pypi.org/project/zenml/)
 
-**Problem statement**: For a given customer's historical data, we are asked to predict whether a customer will stop using a companies product or not. We will be using [Telco Customer Churn](https://www.kaggle.com/datasets/blastchar/telco-customer-churn?datasetId=13996&sortBy=voteCount) dataset for building an end to end production-grade machine learning system that can predict whether the customer will stay loyal or not. The dataset has 20 input features and a target variable for 7043 customers.
+**Problem statement**: For a given customer's historical data, we are asked to predict whether a customer will stop using a company's product or not. We will be using the [Telco Customer Churn](https://www.kaggle.com/datasets/blastchar/telco-customer-churn?datasetId=13996&sortBy=voteCount) dataset for building an end to end production-grade machine learning system that can predict whether the customer will stay loyal or not. The dataset has 20 input features and a target variable for 7043 customers.
 
-Customer churn is a tendency of customers to leave or churn a company and stop paying customers for a particular business. We can calculate a churn rate for a company by dividing the number of customers who churned by the total number of customers and then multiplying that number by 100 per cent. If you want to learn more about customer churn, you can read this[Wikipedia article](https://en.wikipedia.org/wiki/Churn_rate).
+When someone leaves a company and when that customer stops paying a business for its services or products, we call that 'churn'. We can calculate a churn rate for a company by dividing the number of customers who churned by the total number of customers and then multiplying that number by 100 to reach a percentage value. If you want to learn more about customer churn, you can read this [Wikipedia article](https://en.wikipedia.org/wiki/Churn_rate).
 
 To achieve this in a real-world scenario, we will be using [ZenML](https://zenml.io/) to build a production-ready pipeline that predicts whether a customer will churn or not ahead of time.
 The purpose of this repository is to demonstrate how [ZenML](https://github.com/zenml-io/zenml) empowers your business to build and deploy machine learning pipelines in a multitude of ways:
 
 - By offering you a framework or template to develop within.
-- By integrating with popular tools like Kubeflow, Seldon-core, facets, and more.
-- By allowing you to build and deploy your machine learning pipelines easily using the modern MLOps Framework.
+- By integrating with popular and useful tools like Kubeflow, Seldon Core, `facets`, and more.
```

Comment:
If you're removing facets, then you should remove it from the README.

Reply:
No, I am not removing facets, I am using it in data analysis pipeline, I just removed facets from requirements file which was not required.